qmlui/virtualconsole/vccuelist.cpp

```diff
@@ -259,13 +259,14 @@ void VCCueList::setSideFaderLevel(int level)
         int newStep = level; // by default we assume the Chaser has more than 256 steps
         if (ch->stepsCount() < 256)
         {
-            float stepSize = 255 / float(ch->stepsCount());
-            if (level >= 255 - stepSize)
+            float stepSize = 256 / float(ch->stepsCount()); //divide up the full 0..255 range
+            stepSize = (float)qFloor((stepSize * 100000) + 0.5) / 100000; //round to 5 decimals to fix corner cases
```

Comment:
stepSize is already float, so no need to cast it to float again

Reply:
Yep -- but the result of calling qFloor is an int, and the resulting stepSize was getting truncated.  But per your request below, I changed "100000" to "100000.0" and removed the now-unnecessary casts to float.